website/project/views/contributor.py

```diff
@@ -140,63 +138,6 @@ def get_contributors_from_parent(auth, node, **kwargs):
     return {'contributors': contribs}
 
 
-@must_be_contributor_or_public
-def get_most_in_common_contributors(auth, node, **kwargs):
-    node_contrib_ids = set(node.contributors._to_primary_keys())
-    try:
-        n_contribs = int(request.args.get('max', None))
-    except (TypeError, ValueError):
-        n_contribs = settings.MAX_MOST_IN_COMMON_LENGTH
```

Comment:
Can you remove `settings.MAX_MOST_IN_COMMON_LENGTH` if it's no longer being used?